package/database/sqlinstance/composition.yaml

```diff
@@ -15,29 +15,41 @@ spec:
         apiVersion: rds.aws.upbound.io/v1beta1
         kind: SubnetGroup
         spec:
+          deletionPolicy: Delete
```

Comment:
should be removed now that deletionPolicy is being set in the XRD.

Reply:
Removed.